### PR TITLE
[APPC-4187] Refactor duplicated credit card / preselected card layout

### PR DESCRIPTION
**Pull request by rodolfobcardoso**
**What does this PR do?**

   A few sentences describing the overall goals of the pull request's commits.

   Explain any technical decisions that were made and why (when it makes sense).

**Database changed?**

   Yes | No

**Where should the reviewer start?**

- [ ] Foo.java
- [ ] Foo.kt

**How should this be manually tested?**

  Flow on how to test this.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4187](https://aptoide.atlassian.net/browse/APPC-4187)

**Code Review Checklist**

- [ ] Architecture
- [ ] Documentation on public interfaces
- [ ] Database changed?
- [ ] Database migration (if applicable)?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] Functional QA tests pass

[APPC-4187]: https://aptoide.atlassian.net/browse/APPC-4187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by rodolfobcardoso** [comment]
run-jenkins-build

File: app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt [hunk 1]
```diff
@@ -116,182 +119,145 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
...
-  private val bindingCreditCardPreSelected: AdyenCreditCardPreSelectedBinding? by lazy {
-    if (isPreSelected) AdyenCreditCardPreSelectedBinding.bind(
-      requireView()
-    ) else null
-  }
   private val bindingCreditCardLayout: AdyenCreditCardLayoutBinding? by lazy {
-    if (!isPreSelected) AdyenCreditCardLayoutBinding.bind(
-      requireView()
-    ) else null
+    AdyenCreditCardLayoutBinding.bind(requireView())
```
Comment (giovanni-aptoide) [inline app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt:124]: To keep the same pattern of the rest of the app, I would suggest change "bindingCreditCardLayout" just to "binding"

File: app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt [hunk 2]
```diff
@@ -116,182 +119,145 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
...
-  private val buy_button: WalletButtonView
-    get() = bindingCreditCardPreSelected?.dialogBuyButtonsPaymentMethods?.buyButton
-      ?: bindingCreditCardLayout?.dialogBuyButtons?.buyButton!!
-  private val cancel_button: WalletButtonView
-    get() = bindingCreditCardPreSelected?.dialogBuyButtonsPaymentMethods?.cancelButton
-      ?: bindingCreditCardLayout?.dialogBuyButtons?.cancelButton!!
+  private val buyButton: WalletButtonView
+    get() = bindingCreditCardLayout?.dialogBuyButtons?.buyButton!!
+  private val cancelButton: WalletButtonView
+    get() = bindingCreditCardLayout?.dialogBuyButtons?.cancelButton!!
```
Comment (giovanni-aptoide) [inline app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt:132]: Once we don’t have two layouts in the same fragment anymore, I don’t see why to keep variables, why don’t call it directly as the rest of the app, like binding.dialogBuyButtons.cancelButton

File: app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt
Comment (giovanni-aptoide) [inline app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt:1]: this separation of concerns is much better, congrats

**Pull request merged**